packages/types/src/context.ts

```diff
@@ -119,3 +119,8 @@ export interface CloudResourceContext extends Record<string, unknown> {
 export interface ProfileContext extends Record<string, unknown> {
   profile_id: string;
 }
+
+export interface MissingInstrumentationContext extends Record<string, unknown> {
+  package: string;
+  ['javascript.is_cjs']?: boolean;
+}
```

Comment:
Hm, I'd think we would also want to move the types out of this file.

Reply:
Why?

Reply:
Actually, never mind I confused it with the context integration 🫠.